README.md

```
@@ -5,6 +5,27 @@
 
 ---
 
+## Sponsors
+
+We gratefully acknowledge support from our sponsors:
+
+<div align="center">
+  <a href="https://stytch.com">
+    <picture>
+      <source width="200px" media="(prefers-color-scheme: dark)" srcset="./sponsor/stytch-light.svg">
```

markdownlint-cli2 (0.17.2)

Inline HTML
Element: source

(MD033, no-inline-html)